### PR TITLE
Update gtk-sys and make some signatures more precise accordingly

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

File: gtk-sys/build.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@ fn main() {
     for cfg in &cfgs {
         println!("cargo:rustc-cfg={}", cfg);
     }
-    println!("cargo:cfg={}", cfgs.connect(" "));
+    println!("cargo:cfg={}", cfgs.join(" "));
```
Comment (GuillaumeGomez) [inline gtk-sys/build.rs]: I think my PR is still open for this XD

**Comment by GuillaumeGomez** [comment]
Looks good to me. I wait for travis and stuff and then I merge. Thanks for your work !

File: gtk-sys/build.rs (hunk 1 above)
Comment (gkoz) [inline gtk-sys/build.rs]: That PR seems to have been transformed into something entirely different :)

**Comment by gkoz** [comment]
Whatever happened to appveyor. It might not have gotten the memo when the org was renamed. You may need to readd the repos there.

**Pull request merged**